contributor/Development_Workflow.md

```diff
@@ -0,0 +1,26 @@
+# Development Workflow
+
+## Prerequisites
+The OpenLab CI is built based on `NodePool` and `Zuul` tools of the OpenStack infrastructure. The CI Jobs definition uses Ansible playbooks. Learn more about the tools first:
+* Zuul documentation is hosted on [Zuul](https://zuul-ci.org)
+* NodePool documentation is hosted on [NodePool](https://zuul-ci.org/docs/nodepool/)
+* Ansible documentation is hosted on [Ansible Docs](https://docs.ansible.com/)
+
+## Working on Ocean request
+The accepted test request items can be found on [TODO List](https://github.com/orgs/theopenlab/projects/1#column-1860008).
+1. Pick one item you wish to work on, move it to [IN PROGRESS List](https://github.com/orgs/theopenlab/projects/1#column-1860011), edit the item to add the development plan
+2. The target project of the request item should install the **OpenLab-CI** [github APP](https://github.com/apps/theopenlab-ci), so that OpenLab CI system and target project can communicate with each other. See the details on [connecting guideline](./Connect_to_OpenLab.md)
+3. Fork your own copy of the CI Job repository [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs) to your account, develop the test job and submit a PR(Pull Request). In order to manage the test request well, please relate the request number in PR's commit message, the format like:
```

Comment:
1. Fork OpenLab CI Jobs repository [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs) to your account
2. submit a PR(Pull Request) to [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs)
3. please add change details and link the test request number in PR's commit message

Reply:
addressed 2 & 3.